src/tests/subcommands/update_test.go

```diff
@@ -0,0 +1,158 @@
+package tests
```

Comment:
We should actually be adding unit tests to the same folder as the code under test, so this should be in src/subcommands/ - no need to move all the unit tests we've already added to wrong folder in this PR (although it'd be great to do that after this) but we should at least starting putting new unit test files in the right place

Reply:
Yeah I have kept this in the tests folder just to be with the other test files. But will move all the test files to the same folder as the code files in another PR